HTTP2-Core.package/H2WebClient.class/instance/sslConnect.to..st

```diff
@@ -0,0 +1,8 @@
+accessing
+sslConnect: stream to: lastServer
+	"Do the SSL handshake, use SNI if available"
+	(stream respondsTo: #sslConnectTo:withAlpnProtos: )
+		ifTrue: [stream sslConnectTo: lastServer withAlpnProtos: (Set  with: 'h2' with: 'http/1.1').
```

Comment:
We might want to move `Set  with: 'h2' with: 'http/1.1'` to another method since it is a "magic" value